lib/galaxy/datatypes/converters/fastqcssangergz_to_fastqcssanger.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="CONVERTER_fastqcssangergz_to_fastqcssanger" name="Convert fastqcssanger.gz files to fastqcssanger" version="1.0.0" hidden="true">
+  <command>gzip -dcf $input1 > $output1</command>
```

Comment:
I understand, and normally I agree, but it's not so easy to get a requirement for gzip.